packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -526,6 +526,7 @@ export const activeMeetingsByMeetingSeriesId = (parent: RootDataLoader) => {
             .table('NewMeeting')
             .getAll(key, {index: 'meetingSeriesId'})
             .filter({endedAt: null}, {default: true})
+            .orderBy(r.asc('createdAt'))
```

Comment:
added this to make sure we have a stable meeting order in the response, ie. the newest meeting should be first in the list

Reply:
+1 This should be `r.desc` right? I.e. if `createdAt` is in ascending order, the first meeting will be the oldest.

Reply:
I probably used the wrong words here but what I had in mind was to have the closest active meeting first. we currently don't support multiple instances opened at the same time, but if we ever change that, having `asc` order here makes the most sense to me. Imagine the following scenario

The meeting series has three active meetings - the first one ending tomorrow, the second one ending the day after tomorrow, and the last one ending the day after the second one. I'd say the link to `current activity` should take me to the first one, as other meetings are in the future. That's why I used `asc` here to have the `current` one as the first elem by default. In general, there are no good or bad answers here, it's only a question how define business requirements. For now, I'd leave it as is. @jmtaber129 lmk if that makes sense